src/domain/challenge/space/pages/SpaceCommunity/useSpaceCommunityContext.ts

```diff
@@ -62,6 +67,14 @@ const useSpaceCommunityContext = (spaceId: string) => {
     ),
   };
 
+  const { data: spaceDisplayNameData } = useSpaceDisplayNameQuery({
```

Comment:
Another query just for the displayName sounds excesive to me, isn't there a better way to obtain it or isn't it easy to pass it from higher... in SpaceCommunityPage you have a useSpace which is not nice, but has the profile.